Packs/Okta/Integrations/Okta_IAM/Okta_IAM.py

```diff
@@ -0,0 +1,397 @@
+        causes += f'{idx}. {cause.get("errorSummary")}\n'
+    if causes:
+        error_msg += f'Reason:\n{causes}'
+    return error_msg
+
+
+'''COMMAND FUNCTIONS'''
+
+
+def test_module(client):
+    client.test()
+    return_results('ok')
+
+
+def get_user_command(client, args, mapper_in):
+    user_profile = IAMUserProfile(user_profile=args.get('user-profile'))
+    try:
+        okta_user = client.get_user(user_profile.get_attribute('email'))
+        if not okta_user:
+            error_code, error_message = IAMErrors.USER_DOES_NOT_EXIST
+            user_profile.set_result(action=IAMActions.GET_USER,
+                                    success=False,
+                                    error_code=error_code,
+                                    error_message=error_message)
+        else:
+            user_profile.update_with_app_data(okta_user, mapper_in)
+            user_profile.set_result(
+                action=IAMActions.GET_USER,
+                success=True,
+                active=False if okta_user.get('status') == DEPROVISIONED_STATUS else True,
+                iden=okta_user.get('id'),
+                email=okta_user.get('profile', {}).get('email'),
+                username=okta_user.get('profile', {}).get('login'),
+                details=okta_user
+            )
+
+    except DemistoException as e:
+        handle_exception(user_profile, e, IAMActions.GET_USER)
+
+    return user_profile
+
+
+def enable_user_command(client, args, mapper_out, is_command_enabled, is_create_user_enabled, create_if_not_exists):
+    user_profile = IAMUserProfile(user_profile=args.get('user-profile'))
+    if not is_command_enabled:
+        user_profile.set_result(action=IAMActions.ENABLE_USER,
+                                skip=True,
+                                skip_reason='Command is disabled.')
+    else:
+        try:
+            okta_user = client.get_user(user_profile.get_attribute('email'))
+            if not okta_user:
+                if create_if_not_exists:
+                    user_profile = create_user_command(client, args, mapper_out, is_create_user_enabled)
+                else:
+                    _, error_message = IAMErrors.USER_DOES_NOT_EXIST
+                    user_profile.set_result(action=IAMActions.ENABLE_USER,
+                                            skip=True,
+                                            skip_reason=error_message)
+            else:
+                client.activate_user(okta_user.get('id'))
+                user_profile.set_result(
+                    action=IAMActions.ENABLE_USER,
+                    success=True,
+                    active=True,
+                    iden=okta_user.get('id'),
+                    email=okta_user.get('profile', {}).get('email'),
+                    username=okta_user.get('profile', {}).get('login'),
+                    details=okta_user
+                )
+
+        except DemistoException as e:
```

Comment:
add also `expect Exception as e1` we should also catch other exceptions other than DemistoException. Otherwise the whole command will fail, the UserProfile will not be updated via errorCode and errorMessage